app/assets/stylesheets/modules/content.scss

```diff
@@ -0,0 +1,46 @@
+@import "foundation_and_overrides";
+@import "globals/all";
+
+$content-padding: rem-calc(50);
+
+.content {
+  @include panel(
+    $bg: $white
+  );
+  border-radius: $global-radius;
+  margin-top: $global-vertical-spacing;
+  overflow: hidden;
+  padding: 0 $content-padding $content-padding $content-padding !important;
```

Comment:
!important is used to override Foundation—a third-party framework. For this reason, it's OK.